CodeForAfrica-SCRAPERS/taxclock_ke_standard
# Ch send messages

**What does this PR do?**
Sends slack messages to slack in case of errors.
**How can this be tested?** 
Run the command `python scraper.py`
Description:
Implemented the functionality using  slacker-log-handler 
There is need to configure the slack channel  and channel token the messages should be send to.

**Sample slack message.**
<img width="1086" alt="screen shot 2017-09-22 at 5 35 49 pm" src="https://user-images.githubusercontent.com/27199863/30749593-8bd41c70-9fbc-11e7-8f2b-29c760b00632.png">
**Sample LogFile.**
<img width="929" alt="screen shot 2017-09-25 at 2 57 20 pm" src="https://user-images.githubusercontent.com/27199863/30807334-0ad63ade-a202-11e7-8222-c1e4410329ad.png">
